sync/class.jetpack-sync-full.php

```diff
@@ -258,22 +248,22 @@ public function expand_term_ids( $args ) {
 
 	public function expand_options( $args ) {
```

Comment:
I started doing that, but it felt like something we should tackle in a separate PR. It's actually a bunch of work because there's some shared code around efficiently serializing/expanding object ID's which it's useful to keep in the same file.

But yeah, I do think we should find a design that lets us move some of these sync-content-specific functions into the appropriate modules.